damnit/gui/plot.py

```diff
@@ -35,6 +38,9 @@ def __init__(
     ):
         super().__init__()
         self.setStyleSheet("background-color: white")
+        self.setStyleSheet("::selection{ background-color: blue }")
```

Comment:
I added this line since the line above was overwriting the background color for all elements (including the pseudo-element `selection`). A easier fix is to rewrite the line above using the QDialog selector.